GraphWebApi/Controllers/GraphExplorerAppModeController.cs

```diff
@@ -145,25 +144,32 @@ private async Task<IActionResult> ProcessRequestAsync(string method, string all,
             }
 
             var contentType = "application/json";
-            try
+           
+            using (var response = await request.SendRequestAsync(content?.ToString(), CancellationToken.None, HttpCompletionOption.ResponseContentRead).ConfigureAwait(false))
             {
-                using (var response = await request.SendRequestAsync(content?.ToString(), CancellationToken.None, HttpCompletionOption.ResponseContentRead).ConfigureAwait(false))
-                {
-                    response.Content.Headers.TryGetValues("content-type", out var contentTypes);
+                response.Content.Headers.TryGetValues("content-type", out var contentTypes);
 
-                    contentType = contentTypes?.FirstOrDefault() ?? contentType;
+                contentType = contentTypes?.FirstOrDefault() ?? contentType;
 
-                    var byteArrayContent = await response.Content.ReadAsByteArrayAsync().ConfigureAwait(false);
-                    Console.WriteLine(byteArrayContent);
-                    return new HttpResponseMessageResult(ReturnHttpResponseMessage(HttpStatusCode.OK, contentType, new ByteArrayContent(byteArrayContent)));
-                }
-            }
-            catch (ServiceException ex)
-            {
-                return new HttpResponseMessageResult(ReturnHttpResponseMessage(ex.StatusCode, contentType, new StringContent(ex.Error.ToString())));
+                var byteArrayContent = await response.Content.ReadAsByteArrayAsync().ConfigureAwait(false);
+                Console.WriteLine(byteArrayContent);
```

Comment:
Oops, didn't see this earlier. Can this  console.writeline be deleted with this pr